terraform/infrastructure/oqtopus-prod/variables.tf

```diff
@@ -27,3 +27,27 @@ variable "profile" {
   description = "aws profile"
   type        = string
 }
+
+variable "vpc_cidr" {
+  description = "The CIDR block for the VPC"
+  type        = string
+  default     = "10.2.0.0/16"
+}
+
+variable "availability_zones" {
+  description = "List of availability zones"
+  type        = list(string)
+  default     = ["a", "c", "d"]
+}
+
+variable "private_subnet_cidrs" {
+  description = "CIDR blocks for private subnets"
+  type        = list(string)
+  default     = ["10.2.128.0/20", "10.2.144.0/20", "10.2.160.0/20"]
+}
+
+variable "public_subnet_cidr" {
+  description = "CIDR block for public subnet"
+  type        = string
+  default     = "10.2.176.0/20"
+}
```

Comment:
@massn 
See following comment.
https://github.com/oqtopus-team/oqtopus-cloud/pull/247/files#r2173149518

Reply:
Added them to `oqtopus-dev`